.github/workflows/upload-release.yml

```
@@ -0,0 +1,69 @@
+        default: false
+permissions:
+  contents: read
+  id-token: write
+  packages: write
+jobs:
+  release:
+    runs-on: ubuntu-24.04
+    environment: releases
+    steps:
+      - uses: actions/checkout@8e8c483db84b4bee98b60c0593521ed34d9990e8 # v6.0.1
+      - uses: ./.github/actions/install-nix-action
```

actionlint (1.7.10)

missing input "github_token" which is required by action "Install Nix" defined at "./.github/actions/install-nix-action". all required inputs are "dogfood", "github_token"

(action)